middleware/configuration/documentation/domain.general.operation.md

```diff
@@ -149,7 +149,6 @@ domain:
         timeout:
```

Comment:
Github wont let me comment on the right lines, but on 53 & 132 discoverable is listed as a property, but it's now called 'visibility', right?

Reply:
correct, good catch!

Reply:
I'll fix that. But you're ok with the rest?

Reply:
Yeah, looks good to me